go/apps/api/openapi/openapi.yaml

```diff
@@ -635,7 +334,7 @@ components:
         meta:
           "$ref": "#/components/schemas/Meta"
         data:
-          "$ref": "#/components/schemas/KeysGetKeyResponseData"
+          "$ref": "#/components/schemas/KeyResponse"
```

Comment:
I think we should stick to the naming scheme KeyResponseData
I'm guessing you renamed this cause you're reusing it for list keys. I just want to be mindful of these names, cause they will show up in generated sdks etc too and it can get messy quickly

Reply:
actually I'm not sure here, but I do want consistency :D